cleo_plugins/Audio/Audio.cpp

```diff
@@ -8,7 +8,7 @@
 using namespace CLEO;
 using namespace plugin;
 
-#define VALIDATE_STREAM() if(!soundSystem.HasStream(stream)) { SHOW_ERROR("Invalid or already closed '0x%X' audio stream handle param in script %s \nScript suspended.", stream, ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
+#define VALIDATE_STREAM() if(stream != nullptr && !soundSystem.HasStream(stream)) { SHOW_ERROR("Invalid or already closed '0x%X' audio stream handle param in script %s \nScript suspended.", stream, ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
```

Comment:
shouldn't we also suspend when stream is nullptr?
```
if(stream == nullptr || !soundSystem.HasStream(stream))
```

Reply:
Legacy code allowed null ptr as input arg in sound stream opcodes.
These changes are dictated by actuall mod that happens to relay on it.